### ISSUE TITLE
How Can I Avoid Appending Date Until After File Roll

### ISSUE THREAD

**Issue by d8ahazard** [serilog/serilog-sinks-rollingfile#78]
Hello!

I love the rolling file feature, but it creates one minor issue for me that I'm hoping to overcome.

As the title suggest, I'd like to avoid appending the date to the "active" log file until it's no longer in use. So, for example, if my basic log file name is "TestApp.log", then I'd like the file that is actively being written to to always be TestApp.log. Once the file rotates, then the timestamp gets appended.

Basically, my logs currently rotate every hour, which makes it mildly annoying to try tailing a log and then have it cut off every hour. So, I was hoping that maybe there was some kind of option to have this sort of rolling behavior instead.

TIA!

**Comment by cocowalla** [comment]
Hi, thanks for the input!

There is already a closed issue covering this, #26.

This sink has now been deprecated in favour of [serilog-sinks-file](https://github.com/serilog/serilog-sinks-file), so it's unlikely any changes will be made here. I should also add that your suggestion had already been made over in [serilog-sinks-file](https://github.com/serilog/serilog-sinks-file) - there has been a fair bit of interest in this, but it's far from a trivial change to make.
